tests/main/snap-run/task.yaml

```diff
@@ -48,7 +48,7 @@ execute: |
     fi
     MATCH "Can't stat 'invalid': No such file or directory" < stderr
 
-    if [[ "$SPREAD_SYSTEM" == arch-* || "$SPREAD_SYSTEM" == opensuse-tumbleweed-* ]] ; then
+    if os.query is-arch-linux || [[ "$SPREAD_SYSTEM" == opensuse-tumbleweed-* ]] ; then
```

Comment:
should we have `os.query is-opensuse-tumbleweed` ?

Reply:
yes, but it is going to be included in next PR, a command for more systems versions